tests/e2e_pushdown/test_sql_safety_sweep.py

```
@@ -0,0 +1,62 @@
+import pytest
+
+from federated_query.parser.errors import UnsupportedSQLError
+
+from tests.e2e_pushdown.helpers import build_runtime
+
+TABLE = "duckdb_primary.main.orders"
+
+
+# Each case is a SQL fragment the engine must reject rather than misinterpret.
+REJECTED_SQL = [
+    f"SELECT order_id FROM {TABLE} ORDER BY price DESC FETCH FIRST 2 ROWS WITH TIES",
```

Ruff (0.15.18)

[error] Possible SQL injection vector through string-based query construction

(S608)